lang/java/reef-bridge-client/src/main/avro/JobSubmissionParameters.avsc

```diff
@@ -60,6 +60,7 @@
         { "name": "securityTokenKind", "type": "string", "default": "NULL" },
         { "name": "securityTokenService", "type": "string", "default": "NULL" },
         { "name": "driverMemory", "type": "int" },
+        {"name": "envMap", "type": {"type": "map", "values": "string"}},
```

Comment:
Consider `environmentVariablesMap` to make the name more self-documenting.

Reply:
Done